plugins/GSdx/res/glsl/tfx_fs.glsl

```diff
@@ -860,12 +860,28 @@ void ps_main()
         // C.rgb = (C.rgb - 256.0f);
     // }
 // #endif
+#if PS_SCANMSK_TRANSPARENT
+    alpha_blend /= 2.0f;
+#endif
     SV_Target0 = C / 255.0f;
     SV_Target1 = vec4(alpha_blend);
 
 #if PS_ZCLAMP
 	gl_FragDepth = min(gl_FragCoord.z, MaxDepthPS);
-#endif 
+#else
+ 	gl_FragDepth = gl_FragCoord.z;
```

Comment:
This is a terrible idea. I understand you did this because you wanted it to make the thing fail the depth test to discard it (and if you write depth in a branch you have to always write it), but even a normal discard probably would have been better. This code makes everything slower to the benefit of a single game.

Reply:
Ah, I see, didn't know about discard. Changed that now.